docs/UserGuide.md

```diff
@@ -17,28 +17,48 @@ MindMap is a **desktop app for managing contacts, optimized for use via a  Line
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
-
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
-
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-W13-3/tp/releases).
+
+1. Copy the file to the folder you want to use as the _home folder_ for your MindMap application.
```

Comment:
seems to be a mistake in the numbering with multiple 1s

Reply:
Not a mistake. MarkDown converts the `1`s into proper steps.